android/KMEA/app/src/test/java/com/tavultesoft/kmea/logic/ResourcesUpdateToolTest.java

```diff
@@ -53,7 +53,7 @@ public void shouldUpdateFalseCallTest2() {
   @Test
   public void shouldUpdateTrueCallTest() {
     Calendar _cal = GregorianCalendar.getInstance();
-    _cal.add(Calendar.DAY_OF_MONTH,-1);
+    _cal.add(Calendar.DAY_OF_MONTH,-2);
```

Comment:
The "24 hour" window the unit test relies on for determining whether to update resources fails when the CI server goes through Daylight Savings time change :/